crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -229,7 +229,9 @@ impl ExtractComponent for BloomSettings {
                     viewport: UVec4::new(origin.x, origin.y, size.x, size.y).as_vec4()
                         / UVec4::new(target_size.x, target_size.y, target_size.x, target_size.y)
                             .as_vec4(),
-                    aspect: AspectRatio::from_pixels(size.x, size.y).into(),
+                    aspect: AspectRatio::try_from_pixels(size.x, size.y)
+                        .unwrap()
```

Comment:
Do we know if at these points the values going in will be valid? Maybe we should opt for `.expect` instead with a more useful panic message. Like in this case: `.expect("Valid screen size values for Bloom settings")`.

Reply:
ah.. sorry. I must have made a mistake because I just woke up.